apps/vue-form-builder/src/types/field.types.ts

```diff
@@ -264,6 +264,14 @@ export const FIELD_INSTANCE_PROPS_KEYS = [
  * The properties of a field instance that are spread into the component.
  * This can be aded to the component props definition to make the field instance available in the component.
  */
-export type FieldInstanceProps = Partial<
-  UnwrapNestedRefs<Pick<FieldInstance, (typeof FIELD_INSTANCE_PROPS_KEYS)[number]>>
->;
+export type FieldInstanceProps = {
+  name?: string;
```

Comment:
```suggestion
  name: string;
```

Denk dat deze gewoon verplicht moet zijn.

Reply:
Dat had ik eerst ook alleen als je deze prop dus gebruikt met `FieldInstanceProps & {}` en je component heeft niet al deze props dan krijg je daar errors van dat lijkt mij ook niet de bedoeling

Reply:
denk je dat we ook situaties krijgen waar de name niet bekend is?

Reply:
op dit moment zie ik bij de checkbox dat hij geen name heeft en bij andere van onze componenten soms ook al optional

Reply:
ok, goed om hier misschien toch een standaard in te vinden. Want dit is wel het moment om zulke dingen typewise af te dwingen.

Al met al, wellicht toe verplichten in de toekomst.

Mijn zegen heb je ✅

Reply:
Ja je zou het in de toekomst met een breaking change kunnen aanpassen als we dat wel willen. Je kan natuurlijk altijd in je eigen props nog required maken